src/transformers/models/deberta_v2/modeling_deberta_v2.py

```diff
@@ -766,7 +766,7 @@ def disentangled_attention_bias(self, query_layer, key_layer, relative_pos, rel_
 
         score = 0
         # content->position
-        if "c2p" in self.pos_att_type:
+        if "c2p" in self.pos_att_type or "p2p" in self.pos_att_type:
```

Comment:
This line should be keep unchanged. Actually p2p is not used in our current public model.

Reply:
If we keep this line unchanged I suggest removing the p2p support entirely (as it is the case for Deberta). As otherwise the variable `c2p_pos` used by the `p2p` attention would not be initialized. I will push an update to this PR make these changes.